File: src/main/java/no/nav/syfo/domain/InngaendeJournalpost.java
package no.nav.syfo.domain;

import lombok.Builder;
import lombok.Getter;
import lombok.Value;

import java.util.Optional;

@Value
@Builder
@Getter
public class InngaendeJournalpost {
    String fnr;
    String journalpostId;
    String dokumentId;
    String behandlendeEnhetId;
    String gsakId;
    Optional<String> arbeidsgiverOrgnummer;
    Optional<String> orgnummerPrivatperson;
}

File: syfoinntektsmelding-core/src/main/kotlin/no/nav/syfo/domain/InngaendeJournalpost.kt
package no.nav.syfo.domain

data class InngaendeJournalpost(
    val fnr: String,
    val journalpostId: String,
    val dokumentId: String,
    val behandlendeEnhetId: String,
    val gsakId: String,
    val arbeidsgiverOrgnummer: String? = null,
    val arbeidsgiverPrivat: String? = null
)
